UnitTests/MockDefaultValueProviderFixture.cs

```diff
@@ -93,6 +93,15 @@ public void CreatedMockIsVerifiedWithOwner()
 			Assert.Throws<MockVerificationException>(() => mock.Verify());
 		}
 
+		[Fact]
+		public void DefaultValueIsNotChangedWhenPerformingInternalInvocation()
+		{
+			var mockBar = new Mock<IBar> { DefaultValue = DefaultValue.Empty };
```

Comment:
Given that `DefaultValue` is set to `DefaultValue.Empty`, would this test probably be more at home in `EmptyDefaultValueProviderFixture.cs`?